project/src/main/scala/apps/analytics/ModelClassifier.scala

```diff
@@ -0,0 +1,22 @@
+package apps.analytics
+
+import org.semanticweb.HermiT.{ Reasoner => HermiTReasoner }
+
+object ModelClassifier
+{
+
+} // ModelClassifier
+
+/** The `ModelClassifierTest` object contains some tests for the
+ *  `ModelClassifier` class.
+ */ 
+object ModelClassifierTest extends App
```

Comment:
Eventually, we should change this to a concrete example that not only shows whether or not the ontology + added individuals is consistent, but also shows the classes inferred for the model individuals.